cypress/integration/routing.spec.js

```diff
@@ -3,28 +3,28 @@
 // corresponding requirements: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/143
 describe('urls for lemma detail page should be handled correctly', ()=>{
   it('should show lemma detail (paradigms) if a unambiguous url is given', function () {
-
     // Get to the definition/paradigm page for "wâpamêw"
     cy.visit('/word/wâpamêw/')
     cy.get('[data-cy=paradigm]')
       .should('be.visible')
       .and('contain', 'kiwâpamitin')
   })
-  it('should redirect to search page if no match is found', function () {
 
-    // poopoo is a fictional word
-    cy.visit('/word/poopoo/')
+  it('should redirect to search page if no match is found', function () {
+    // pôpô is a fictional word
+    let fakeWord = 'pîpîpôpô'
```

Comment:
@Madoshakalaka I hope you like what I replaced it with

Reply:
![pie](https://user-images.githubusercontent.com/44753941/76366628-26273c00-62f0-11ea-98d4-68152b163026.jpg)